pallets/hsm/src/weights.rs

```diff
@@ -0,0 +1,48 @@
+#![cfg_attr(rustfmt, rustfmt_skip)]
+#![allow(unused_parens)]
+#![allow(unused_imports)]
+#![allow(missing_docs)]
+
+use frame_support::{traits::Get, weights::{Weight, constants::RocksDbWeight}};
+use core::marker::PhantomData;
+
+/// Weight functions needed for pallet_hsm.
+pub trait WeightInfo {
+	fn add_collateral_asset() -> Weight;
+	fn remove_collateral_asset() -> Weight;
+	fn update_collateral_asset() -> Weight;
+	fn sell() -> Weight;
+	fn buy() -> Weight;
+	fn execute_arbitrage() -> Weight;
+	fn on_finalize() -> Weight;
+}
+/// Default weights
+impl WeightInfo for () {
+	fn add_collateral_asset() -> Weight {
+		Weight::zero()
```

Comment:
there should be actual benchmarked values here. dont implement it for () otherwise

Reply:
i want to mark it - only for tests, that means for implementation for () would be available only for tests.

Reply:
then it should be only compiled in test runtime

i dont want this to end up in mainnet runtime by some freak accident